examples/resources/kentik-synthetic_test-bulk-create/agents_filtered_by_name/resource.tf

```diff
@@ -0,0 +1,54 @@
+// Create a test with list of agents that have a name containing given substring.
+// Inputs: name substring
+
+data "kentik-synthetics_agents" "agents" {
+  name_substring = "global-agent"
+}
+
+locals {
+  agents_ids = [for agent in data.kentik-synthetics_agents.agents.items: agent.id]
```

Comment:
Yeah, it seems like a clean solution, maybe I rushed extending data-source on this one.

Reply:
Addressed: 5cdf88ed627acb5551e79851307ec56e06ea82a1

Reply:
@ziemowit141 Have you tried filtering elements with `if` (see: https://www.terraform.io/docs/language/expressions/for.html#filtering-elements)? It would be better. I still think using `compact` here is a hack.

Reply:
@Fenthick, now filtering with if statements: 200432c1130c9411d8b4352b23c8ac6ba8f24d03